packages/web/src/components/form2/Field.tsx

```diff
@@ -618,17 +624,19 @@ export default function Field({
                 >
                   Add Response
                 </Button> */}
-                <Tooltip title="Add Response">
-                  <IconButton
-                    data-testid="response-button"
-                    onClick={() => setAddOption({ ...addOption, showDrawer: true })}
-                    size="small"
-                    color="primary"
-                    sx={{ border: '1px solid' }}
-                  >
-                    <Add />
-                  </IconButton>
-                </Tooltip>
+                {!addOption?.showDrawer && (
```

Comment:
can you explain why you are checking showDrawer is false? who , when , why and where showDrawer value is set?

Reply:
1. We declare the showDrawer as false at first : https://github.com/Codemarket2/boossti-frontend/blob/saichidvi3/packages/web/src/components/form2/Field.tsx#L160
2.  This is the plus icon which is responsible for displaying the form 
https://github.com/Codemarket2/boossti-frontend/blob/saichidvi3/packages/web/src/components/form2/Field.tsx#L629
when ever the button is clicked we will set the state of this showDrawer to true from false .
3 . When showDrawer is true we will not display the plus button because already the user is filling one form , so we should not confuse him by keeping the plus button even before saving the new form .
https://github.com/Codemarket2/boossti-frontend/blob/saichidvi3/packages/web/src/components/form2/Field.tsx#L627
4. then we will disply the form for which user want to create the response 
https://github.com/Codemarket2/boossti-frontend/blob/saichidvi3/packages/web/src/components/form2/Field.tsx#L600